packages/jest-haste-map/src/types.js

```diff
@@ -16,13 +16,17 @@ export type IgnoreMatcher = (item: string) => boolean;
 
 export type WorkerMessage = {
   filePath: string,
+  hasteImplModulePath?: string,
 };
 export type WorkerMetadata = {
   id: ?string,
   module: ?ModuleMetaData,
   dependencies: ?Array<string>,
 };
-export type WorkerCallback = (error: ?SerializableError, metaData: ?WorkerMetadata) => void;
+export type WorkerCallback = (
```

Comment:
I broke the build by introducing a lint error in #2875. Sorry! Let me know if you'd like to split this off into another pull request.